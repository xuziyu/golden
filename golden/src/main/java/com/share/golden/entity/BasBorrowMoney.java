package com.share.golden.entity;

public class BasBorrowMoney {
    private String borrowId;

    private String borrowSerial;

    private String userId;

    private String borrowCompany;

    private String borrowMoney;

    private String borrowRate;

    private String timeLimit;

    private String borrowLimit;

    private String borrowState;

    private String borrowRecommend;

    private String borrowType;

    private String borrowRemark1;

    private String borrowRemark2;

    public String getBorrowId() {
        return borrowId;
    }

    public void setBorrowId(String borrowId) {
        this.borrowId = borrowId == null ? null : borrowId.trim();
    }

    public String getBorrowSerial() {
        return borrowSerial;
    }

    public void setBorrowSerial(String borrowSerial) {
        this.borrowSerial = borrowSerial == null ? null : borrowSerial.trim();
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId == null ? null : userId.trim();
    }

    public String getBorrowCompany() {
        return borrowCompany;
    }

    public void setBorrowCompany(String borrowCompany) {
        this.borrowCompany = borrowCompany == null ? null : borrowCompany.trim();
    }

    public String getBorrowMoney() {
        return borrowMoney;
    }

    public void setBorrowMoney(String borrowMoney) {
        this.borrowMoney = borrowMoney == null ? null : borrowMoney.trim();
    }

    public String getBorrowRate() {
        return borrowRate;
    }

    public void setBorrowRate(String borrowRate) {
        this.borrowRate = borrowRate == null ? null : borrowRate.trim();
    }

    public String getTimeLimit() {
        return timeLimit;
    }

    public void setTimeLimit(String timeLimit) {
        this.timeLimit = timeLimit == null ? null : timeLimit.trim();
    }

    public String getBorrowLimit() {
        return borrowLimit;
    }

    public void setBorrowLimit(String borrowLimit) {
        this.borrowLimit = borrowLimit == null ? null : borrowLimit.trim();
    }

    public String getBorrowState() {
        return borrowState;
    }

    public void setBorrowState(String borrowState) {
        this.borrowState = borrowState == null ? null : borrowState.trim();
    }

    public String getBorrowRecommend() {
        return borrowRecommend;
    }

    public void setBorrowRecommend(String borrowRecommend) {
        this.borrowRecommend = borrowRecommend == null ? null : borrowRecommend.trim();
    }

    public String getBorrowType() {
        return borrowType;
    }

    public void setBorrowType(String borrowType) {
        this.borrowType = borrowType == null ? null : borrowType.trim();
    }

    public String getBorrowRemark1() {
        return borrowRemark1;
    }

    public void setBorrowRemark1(String borrowRemark1) {
        this.borrowRemark1 = borrowRemark1 == null ? null : borrowRemark1.trim();
    }

    public String getBorrowRemark2() {
        return borrowRemark2;
    }

    public void setBorrowRemark2(String borrowRemark2) {
        this.borrowRemark2 = borrowRemark2 == null ? null : borrowRemark2.trim();
    }
}