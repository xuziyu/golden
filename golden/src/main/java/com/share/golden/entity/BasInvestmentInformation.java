package com.share.golden.entity;

import java.math.BigDecimal;
import java.util.Date;

public class BasInvestmentInformation {
    private String investorId;

    private String userId;

    private String tenderId;

    private BigDecimal investorMoney;

    private String investorStatus;

    private String investorStyle;

    private String brrowStatus;

    private String investorRest;

    private String profitModel;

    private BigDecimal profitMoney;

    private Date investorDate;

    private String replayDate;

    private String investorType;

    public String getInvestorId() {
        return investorId;
    }

    public void setInvestorId(String investorId) {
        this.investorId = investorId == null ? null : investorId.trim();
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId == null ? null : userId.trim();
    }

    public String getTenderId() {
        return tenderId;
    }

    public void setTenderId(String tenderId) {
        this.tenderId = tenderId == null ? null : tenderId.trim();
    }

    public BigDecimal getInvestorMoney() {
        return investorMoney;
    }

    public void setInvestorMoney(BigDecimal investorMoney) {
        this.investorMoney = investorMoney;
    }

    public String getInvestorStatus() {
        return investorStatus;
    }

    public void setInvestorStatus(String investorStatus) {
        this.investorStatus = investorStatus == null ? null : investorStatus.trim();
    }

    public String getInvestorStyle() {
        return investorStyle;
    }

    public void setInvestorStyle(String investorStyle) {
        this.investorStyle = investorStyle == null ? null : investorStyle.trim();
    }

    public String getBrrowStatus() {
        return brrowStatus;
    }

    public void setBrrowStatus(String brrowStatus) {
        this.brrowStatus = brrowStatus == null ? null : brrowStatus.trim();
    }

    public String getInvestorRest() {
        return investorRest;
    }

    public void setInvestorRest(String investorRest) {
        this.investorRest = investorRest == null ? null : investorRest.trim();
    }

    public String getProfitModel() {
        return profitModel;
    }

    public void setProfitModel(String profitModel) {
        this.profitModel = profitModel == null ? null : profitModel.trim();
    }

    public BigDecimal getProfitMoney() {
        return profitMoney;
    }

    public void setProfitMoney(BigDecimal profitMoney) {
        this.profitMoney = profitMoney;
    }

    public Date getInvestorDate() {
        return investorDate;
    }

    public void setInvestorDate(Date investorDate) {
        this.investorDate = investorDate;
    }

    public String getReplayDate() {
        return replayDate;
    }

    public void setReplayDate(String replayDate) {
        this.replayDate = replayDate == null ? null : replayDate.trim();
    }

    public String getInvestorType() {
        return investorType;
    }

    public void setInvestorType(String investorType) {
        this.investorType = investorType == null ? null : investorType.trim();
    }
}