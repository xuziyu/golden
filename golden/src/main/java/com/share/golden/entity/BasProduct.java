package com.share.golden.entity;

import java.math.BigDecimal;
import java.util.Date;

public class BasProduct {
    private String productId;

    private BigDecimal productMoney;

    private String productIncome;

    private Date productTime;

    private String productType;

    private String productWay;

    private Date pcount;

    private String productProgress;

    private String productSaveway;

    private String productRateben;

    private Date productPublishTime;

    private String productName;

    private BigDecimal productTotalMoney;

    private String investmentScope;

    private String useOfProceeds;

    private String productState;

    private String productPicture;

    private String produceId;

    private String productDesc;

    private String productSafe;

    public String getProductId() {
        return productId;
    }

    public void setProductId(String productId) {
        this.productId = productId == null ? null : productId.trim();
    }

    public BigDecimal getProductMoney() {
        return productMoney;
    }

    public void setProductMoney(BigDecimal productMoney) {
        this.productMoney = productMoney;
    }

    public String getProductIncome() {
        return productIncome;
    }

    public void setProductIncome(String productIncome) {
        this.productIncome = productIncome == null ? null : productIncome.trim();
    }

    public Date getProductTime() {
        return productTime;
    }

    public void setProductTime(Date productTime) {
        this.productTime = productTime;
    }

    public String getProductType() {
        return productType;
    }

    public void setProductType(String productType) {
        this.productType = productType == null ? null : productType.trim();
    }

    public String getProductWay() {
        return productWay;
    }

    public void setProductWay(String productWay) {
        this.productWay = productWay == null ? null : productWay.trim();
    }

    public Date getPcount() {
        return pcount;
    }

    public void setPcount(Date pcount) {
        this.pcount = pcount;
    }

    public String getProductProgress() {
        return productProgress;
    }

    public void setProductProgress(String productProgress) {
        this.productProgress = productProgress == null ? null : productProgress.trim();
    }

    public String getProductSaveway() {
        return productSaveway;
    }

    public void setProductSaveway(String productSaveway) {
        this.productSaveway = productSaveway == null ? null : productSaveway.trim();
    }

    public String getProductRateben() {
        return productRateben;
    }

    public void setProductRateben(String productRateben) {
        this.productRateben = productRateben == null ? null : productRateben.trim();
    }

    public Date getProductPublishTime() {
        return productPublishTime;
    }

    public void setProductPublishTime(Date productPublishTime) {
        this.productPublishTime = productPublishTime;
    }

    public String getProductName() {
        return productName;
    }

    public void setProductName(String productName) {
        this.productName = productName == null ? null : productName.trim();
    }

    public BigDecimal getProductTotalMoney() {
        return productTotalMoney;
    }

    public void setProductTotalMoney(BigDecimal productTotalMoney) {
        this.productTotalMoney = productTotalMoney;
    }

    public String getInvestmentScope() {
        return investmentScope;
    }

    public void setInvestmentScope(String investmentScope) {
        this.investmentScope = investmentScope == null ? null : investmentScope.trim();
    }

    public String getUseOfProceeds() {
        return useOfProceeds;
    }

    public void setUseOfProceeds(String useOfProceeds) {
        this.useOfProceeds = useOfProceeds == null ? null : useOfProceeds.trim();
    }

    public String getProductState() {
        return productState;
    }

    public void setProductState(String productState) {
        this.productState = productState == null ? null : productState.trim();
    }

    public String getProductPicture() {
        return productPicture;
    }

    public void setProductPicture(String productPicture) {
        this.productPicture = productPicture == null ? null : productPicture.trim();
    }

    public String getProduceId() {
        return produceId;
    }

    public void setProduceId(String produceId) {
        this.produceId = produceId == null ? null : produceId.trim();
    }

    public String getProductDesc() {
        return productDesc;
    }

    public void setProductDesc(String productDesc) {
        this.productDesc = productDesc == null ? null : productDesc.trim();
    }

    public String getProductSafe() {
        return productSafe;
    }

    public void setProductSafe(String productSafe) {
        this.productSafe = productSafe == null ? null : productSafe.trim();
    }
}