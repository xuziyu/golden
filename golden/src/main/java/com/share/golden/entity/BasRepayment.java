package com.share.golden.entity;

import java.util.Date;

public class BasRepayment {
    private String repayId;

    private String repayCode;

    private Date repayDate;

    private String repayState;

    private String repayTime;

    private String repayRemark1;

    private String repayRemark2;

    public String getRepayId() {
        return repayId;
    }

    public void setRepayId(String repayId) {
        this.repayId = repayId == null ? null : repayId.trim();
    }

    public String getRepayCode() {
        return repayCode;
    }

    public void setRepayCode(String repayCode) {
        this.repayCode = repayCode == null ? null : repayCode.trim();
    }

    public Date getRepayDate() {
        return repayDate;
    }

    public void setRepayDate(Date repayDate) {
        this.repayDate = repayDate;
    }

    public String getRepayState() {
        return repayState;
    }

    public void setRepayState(String repayState) {
        this.repayState = repayState == null ? null : repayState.trim();
    }

    public String getRepayTime() {
        return repayTime;
    }

    public void setRepayTime(String repayTime) {
        this.repayTime = repayTime == null ? null : repayTime.trim();
    }

    public String getRepayRemark1() {
        return repayRemark1;
    }

    public void setRepayRemark1(String repayRemark1) {
        this.repayRemark1 = repayRemark1 == null ? null : repayRemark1.trim();
    }

    public String getRepayRemark2() {
        return repayRemark2;
    }

    public void setRepayRemark2(String repayRemark2) {
        this.repayRemark2 = repayRemark2 == null ? null : repayRemark2.trim();
    }
}