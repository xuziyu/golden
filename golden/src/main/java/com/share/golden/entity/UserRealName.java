package com.share.golden.entity;

import java.math.BigDecimal;
import java.util.Date;

public class UserRealName {
    private String realId;

    private String userId;

    private String realName;

    private String gender;

    private String identityNum;

    private Date attestationDate;

    private BigDecimal check;

    private Date brithDate;

    private String placeOfDomicile;

    private String theFrontOfIdCard;

    private String theBackOfIdCard;

    private String auditor;

    private String auditorId;

    public String getRealId() {
        return realId;
    }

    public void setRealId(String realId) {
        this.realId = realId == null ? null : realId.trim();
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId == null ? null : userId.trim();
    }

    public String getRealName() {
        return realName;
    }

    public void setRealName(String realName) {
        this.realName = realName == null ? null : realName.trim();
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender == null ? null : gender.trim();
    }

    public String getIdentityNum() {
        return identityNum;
    }

    public void setIdentityNum(String identityNum) {
        this.identityNum = identityNum == null ? null : identityNum.trim();
    }

    public Date getAttestationDate() {
        return attestationDate;
    }

    public void setAttestationDate(Date attestationDate) {
        this.attestationDate = attestationDate;
    }

    public BigDecimal getCheck() {
        return check;
    }

    public void setCheck(BigDecimal check) {
        this.check = check;
    }

    public Date getBrithDate() {
        return brithDate;
    }

    public void setBrithDate(Date brithDate) {
        this.brithDate = brithDate;
    }

    public String getPlaceOfDomicile() {
        return placeOfDomicile;
    }

    public void setPlaceOfDomicile(String placeOfDomicile) {
        this.placeOfDomicile = placeOfDomicile == null ? null : placeOfDomicile.trim();
    }

    public String getTheFrontOfIdCard() {
        return theFrontOfIdCard;
    }

    public void setTheFrontOfIdCard(String theFrontOfIdCard) {
        this.theFrontOfIdCard = theFrontOfIdCard == null ? null : theFrontOfIdCard.trim();
    }

    public String getTheBackOfIdCard() {
        return theBackOfIdCard;
    }

    public void setTheBackOfIdCard(String theBackOfIdCard) {
        this.theBackOfIdCard = theBackOfIdCard == null ? null : theBackOfIdCard.trim();
    }

    public String getAuditor() {
        return auditor;
    }

    public void setAuditor(String auditor) {
        this.auditor = auditor == null ? null : auditor.trim();
    }

    public String getAuditorId() {
        return auditorId;
    }

    public void setAuditorId(String auditorId) {
        this.auditorId = auditorId == null ? null : auditorId.trim();
    }
}