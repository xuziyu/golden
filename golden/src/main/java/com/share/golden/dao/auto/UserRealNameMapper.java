package com.share.golden.dao.auto;

import com.share.golden.entity.UserRealName;
import com.share.golden.entity.UserRealNameExample;
import java.util.List;
import org.apache.ibatis.annotations.Param;

public interface UserRealNameMapper {
    long countByExample(UserRealNameExample example);

    int deleteByExample(UserRealNameExample example);

    int deleteByPrimaryKey(String realId);

    int insert(UserRealName record);

    int insertSelective(UserRealName record);

    List<UserRealName> selectByExample(UserRealNameExample example);

    UserRealName selectByPrimaryKey(String realId);

    int updateByExampleSelective(@Param("record") UserRealName record, @Param("example") UserRealNameExample example);

    int updateByExample(@Param("record") UserRealName record, @Param("example") UserRealNameExample example);

    int updateByPrimaryKeySelective(UserRealName record);

    int updateByPrimaryKey(UserRealName record);
}