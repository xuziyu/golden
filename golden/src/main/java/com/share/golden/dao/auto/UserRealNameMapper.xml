<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
<mapper namespace="com.share.golden.dao.auto.UserRealNameMapper">
  <resultMap id="BaseResultMap" type="com.share.golden.entity.UserRealName">
    <id column="REAL_ID" jdbcType="VARCHAR" property="realId" />
    <result column="USER_ID" jdbcType="VARCHAR" property="userId" />
    <result column="REAL_NAME" jdbcType="VARCHAR" property="realName" />
    <result column="GENDER" jdbcType="VARCHAR" property="gender" />
    <result column="IDENTITY_NUM" jdbcType="VARCHAR" property="identityNum" />
    <result column="ATTESTATION_DATE" jdbcType="TIMESTAMP" property="attestationDate" />
    <result column="CHECK" jdbcType="DECIMAL" property="check" />
    <result column="BRITH_DATE" jdbcType="TIMESTAMP" property="brithDate" />
    <result column="PLACE_OF_DOMICILE" jdbcType="VARCHAR" property="placeOfDomicile" />
    <result column="THE _FRONT_OF_ID_CARD" jdbcType="VARCHAR" property="theFrontOfIdCard" />
    <result column="THE _BACK_OF_ID_CARD" jdbcType="VARCHAR" property="theBackOfIdCard" />
    <result column="AUDITOR" jdbcType="VARCHAR" property="auditor" />
    <result column="AUDITOR_ID" jdbcType="VARCHAR" property="auditorId" />
  </resultMap>
  <sql id="Example_Where_Clause">
    <where>
      <foreach collection="oredCriteria" item="criteria" separator="or">
        <if test="criteria.valid">
          <trim prefix="(" prefixOverrides="and" suffix=")">
            <foreach collection="criteria.criteria" item="criterion">
              <choose>
                <when test="criterion.noValue">
                  and ${criterion.condition}
                </when>
                <when test="criterion.singleValue">
                  and ${criterion.condition} #{criterion.value}
                </when>
                <when test="criterion.betweenValue">
                  and ${criterion.condition} #{criterion.value} and #{criterion.secondValue}
                </when>
                <when test="criterion.listValue">
                  and ${criterion.condition}
                  <foreach close=")" collection="criterion.value" item="listItem" open="(" separator=",">
                    #{listItem}
                  </foreach>
                </when>
              </choose>
            </foreach>
          </trim>
        </if>
      </foreach>
    </where>
  </sql>
  <sql id="Update_By_Example_Where_Clause">
    <where>
      <foreach collection="example.oredCriteria" item="criteria" separator="or">
        <if test="criteria.valid">
          <trim prefix="(" prefixOverrides="and" suffix=")">
            <foreach collection="criteria.criteria" item="criterion">
              <choose>
                <when test="criterion.noValue">
                  and ${criterion.condition}
                </when>
                <when test="criterion.singleValue">
                  and ${criterion.condition} #{criterion.value}
                </when>
                <when test="criterion.betweenValue">
                  and ${criterion.condition} #{criterion.value} and #{criterion.secondValue}
                </when>
                <when test="criterion.listValue">
                  and ${criterion.condition}
                  <foreach close=")" collection="criterion.value" item="listItem" open="(" separator=",">
                    #{listItem}
                  </foreach>
                </when>
              </choose>
            </foreach>
          </trim>
        </if>
      </foreach>
    </where>
  </sql>
  <sql id="Base_Column_List">
    "REAL_ID", "USER_ID", "REAL_NAME", "GENDER", "IDENTITY_NUM", "ATTESTATION_DATE", 
    "CHECK", "BRITH_DATE", "PLACE_OF_DOMICILE", "THE _FRONT_OF_ID_CARD", "THE _BACK_OF_ID_CARD", 
    "AUDITOR", "AUDITOR_ID"
  </sql>
  <select id="selectByExample" parameterType="com.share.golden.entity.UserRealNameExample" resultMap="BaseResultMap">
    select
    <if test="distinct">
      distinct
    </if>
    <include refid="Base_Column_List" />
    from USER_REAL_NAME
    <if test="_parameter != null">
      <include refid="Example_Where_Clause" />
    </if>
    <if test="orderByClause != null">
      order by ${orderByClause}
    </if>
  </select>
  <select id="selectByPrimaryKey" parameterType="java.lang.String" resultMap="BaseResultMap">
    select 
    <include refid="Base_Column_List" />
    from USER_REAL_NAME
    where "REAL_ID" = #{realId,jdbcType=VARCHAR}
  </select>
  <delete id="deleteByPrimaryKey" parameterType="java.lang.String">
    delete from USER_REAL_NAME
    where "REAL_ID" = #{realId,jdbcType=VARCHAR}
  </delete>
  <delete id="deleteByExample" parameterType="com.share.golden.entity.UserRealNameExample">
    delete from USER_REAL_NAME
    <if test="_parameter != null">
      <include refid="Example_Where_Clause" />
    </if>
  </delete>
  <insert id="insert" parameterType="com.share.golden.entity.UserRealName">
    insert into USER_REAL_NAME ("REAL_ID", "USER_ID", "REAL_NAME", 
      "GENDER", "IDENTITY_NUM", "ATTESTATION_DATE", 
      "CHECK", "BRITH_DATE", "PLACE_OF_DOMICILE", 
      "THE _FRONT_OF_ID_CARD", "THE _BACK_OF_ID_CARD", "AUDITOR", 
      "AUDITOR_ID")
    values (#{realId,jdbcType=VARCHAR}, #{userId,jdbcType=VARCHAR}, #{realName,jdbcType=VARCHAR}, 
      #{gender,jdbcType=VARCHAR}, #{identityNum,jdbcType=VARCHAR}, #{attestationDate,jdbcType=TIMESTAMP}, 
      #{check,jdbcType=DECIMAL}, #{brithDate,jdbcType=TIMESTAMP}, #{placeOfDomicile,jdbcType=VARCHAR}, 
      #{theFrontOfIdCard,jdbcType=VARCHAR}, #{theBackOfIdCard,jdbcType=VARCHAR}, #{auditor,jdbcType=VARCHAR}, 
      #{auditorId,jdbcType=VARCHAR})
  </insert>
  <insert id="insertSelective" parameterType="com.share.golden.entity.UserRealName">
    insert into USER_REAL_NAME
    <trim prefix="(" suffix=")" suffixOverrides=",">
      <if test="realId != null">
        "REAL_ID",
      </if>
      <if test="userId != null">
        "USER_ID",
      </if>
      <if test="realName != null">
        "REAL_NAME",
      </if>
      <if test="gender != null">
        "GENDER",
      </if>
      <if test="identityNum != null">
        "IDENTITY_NUM",
      </if>
      <if test="attestationDate != null">
        "ATTESTATION_DATE",
      </if>
      <if test="check != null">
        "CHECK",
      </if>
      <if test="brithDate != null">
        "BRITH_DATE",
      </if>
      <if test="placeOfDomicile != null">
        "PLACE_OF_DOMICILE",
      </if>
      <if test="theFrontOfIdCard != null">
        "THE _FRONT_OF_ID_CARD",
      </if>
      <if test="theBackOfIdCard != null">
        "THE _BACK_OF_ID_CARD",
      </if>
      <if test="auditor != null">
        "AUDITOR",
      </if>
      <if test="auditorId != null">
        "AUDITOR_ID",
      </if>
    </trim>
    <trim prefix="values (" suffix=")" suffixOverrides=",">
      <if test="realId != null">
        #{realId,jdbcType=VARCHAR},
      </if>
      <if test="userId != null">
        #{userId,jdbcType=VARCHAR},
      </if>
      <if test="realName != null">
        #{realName,jdbcType=VARCHAR},
      </if>
      <if test="gender != null">
        #{gender,jdbcType=VARCHAR},
      </if>
      <if test="identityNum != null">
        #{identityNum,jdbcType=VARCHAR},
      </if>
      <if test="attestationDate != null">
        #{attestationDate,jdbcType=TIMESTAMP},
      </if>
      <if test="check != null">
        #{check,jdbcType=DECIMAL},
      </if>
      <if test="brithDate != null">
        #{brithDate,jdbcType=TIMESTAMP},
      </if>
      <if test="placeOfDomicile != null">
        #{placeOfDomicile,jdbcType=VARCHAR},
      </if>
      <if test="theFrontOfIdCard != null">
        #{theFrontOfIdCard,jdbcType=VARCHAR},
      </if>
      <if test="theBackOfIdCard != null">
        #{theBackOfIdCard,jdbcType=VARCHAR},
      </if>
      <if test="auditor != null">
        #{auditor,jdbcType=VARCHAR},
      </if>
      <if test="auditorId != null">
        #{auditorId,jdbcType=VARCHAR},
      </if>
    </trim>
  </insert>
  <select id="countByExample" parameterType="com.share.golden.entity.UserRealNameExample" resultType="java.lang.Long">
    select count(*) from USER_REAL_NAME
    <if test="_parameter != null">
      <include refid="Example_Where_Clause" />
    </if>
  </select>
  <update id="updateByExampleSelective" parameterType="map">
    update USER_REAL_NAME
    <set>
      <if test="record.realId != null">
        "REAL_ID" = #{record.realId,jdbcType=VARCHAR},
      </if>
      <if test="record.userId != null">
        "USER_ID" = #{record.userId,jdbcType=VARCHAR},
      </if>
      <if test="record.realName != null">
        "REAL_NAME" = #{record.realName,jdbcType=VARCHAR},
      </if>
      <if test="record.gender != null">
        "GENDER" = #{record.gender,jdbcType=VARCHAR},
      </if>
      <if test="record.identityNum != null">
        "IDENTITY_NUM" = #{record.identityNum,jdbcType=VARCHAR},
      </if>
      <if test="record.attestationDate != null">
        "ATTESTATION_DATE" = #{record.attestationDate,jdbcType=TIMESTAMP},
      </if>
      <if test="record.check != null">
        "CHECK" = #{record.check,jdbcType=DECIMAL},
      </if>
      <if test="record.brithDate != null">
        "BRITH_DATE" = #{record.brithDate,jdbcType=TIMESTAMP},
      </if>
      <if test="record.placeOfDomicile != null">
        "PLACE_OF_DOMICILE" = #{record.placeOfDomicile,jdbcType=VARCHAR},
      </if>
      <if test="record.theFrontOfIdCard != null">
        "THE _FRONT_OF_ID_CARD" = #{record.theFrontOfIdCard,jdbcType=VARCHAR},
      </if>
      <if test="record.theBackOfIdCard != null">
        "THE _BACK_OF_ID_CARD" = #{record.theBackOfIdCard,jdbcType=VARCHAR},
      </if>
      <if test="record.auditor != null">
        "AUDITOR" = #{record.auditor,jdbcType=VARCHAR},
      </if>
      <if test="record.auditorId != null">
        "AUDITOR_ID" = #{record.auditorId,jdbcType=VARCHAR},
      </if>
    </set>
    <if test="_parameter != null">
      <include refid="Update_By_Example_Where_Clause" />
    </if>
  </update>
  <update id="updateByExample" parameterType="map">
    update USER_REAL_NAME
    set "REAL_ID" = #{record.realId,jdbcType=VARCHAR},
      "USER_ID" = #{record.userId,jdbcType=VARCHAR},
      "REAL_NAME" = #{record.realName,jdbcType=VARCHAR},
      "GENDER" = #{record.gender,jdbcType=VARCHAR},
      "IDENTITY_NUM" = #{record.identityNum,jdbcType=VARCHAR},
      "ATTESTATION_DATE" = #{record.attestationDate,jdbcType=TIMESTAMP},
      "CHECK" = #{record.check,jdbcType=DECIMAL},
      "BRITH_DATE" = #{record.brithDate,jdbcType=TIMESTAMP},
      "PLACE_OF_DOMICILE" = #{record.placeOfDomicile,jdbcType=VARCHAR},
      "THE _FRONT_OF_ID_CARD" = #{record.theFrontOfIdCard,jdbcType=VARCHAR},
      "THE _BACK_OF_ID_CARD" = #{record.theBackOfIdCard,jdbcType=VARCHAR},
      "AUDITOR" = #{record.auditor,jdbcType=VARCHAR},
      "AUDITOR_ID" = #{record.auditorId,jdbcType=VARCHAR}
    <if test="_parameter != null">
      <include refid="Update_By_Example_Where_Clause" />
    </if>
  </update>
  <update id="updateByPrimaryKeySelective" parameterType="com.share.golden.entity.UserRealName">
    update USER_REAL_NAME
    <set>
      <if test="userId != null">
        "USER_ID" = #{userId,jdbcType=VARCHAR},
      </if>
      <if test="realName != null">
        "REAL_NAME" = #{realName,jdbcType=VARCHAR},
      </if>
      <if test="gender != null">
        "GENDER" = #{gender,jdbcType=VARCHAR},
      </if>
      <if test="identityNum != null">
        "IDENTITY_NUM" = #{identityNum,jdbcType=VARCHAR},
      </if>
      <if test="attestationDate != null">
        "ATTESTATION_DATE" = #{attestationDate,jdbcType=TIMESTAMP},
      </if>
      <if test="check != null">
        "CHECK" = #{check,jdbcType=DECIMAL},
      </if>
      <if test="brithDate != null">
        "BRITH_DATE" = #{brithDate,jdbcType=TIMESTAMP},
      </if>
      <if test="placeOfDomicile != null">
        "PLACE_OF_DOMICILE" = #{placeOfDomicile,jdbcType=VARCHAR},
      </if>
      <if test="theFrontOfIdCard != null">
        "THE _FRONT_OF_ID_CARD" = #{theFrontOfIdCard,jdbcType=VARCHAR},
      </if>
      <if test="theBackOfIdCard != null">
        "THE _BACK_OF_ID_CARD" = #{theBackOfIdCard,jdbcType=VARCHAR},
      </if>
      <if test="auditor != null">
        "AUDITOR" = #{auditor,jdbcType=VARCHAR},
      </if>
      <if test="auditorId != null">
        "AUDITOR_ID" = #{auditorId,jdbcType=VARCHAR},
      </if>
    </set>
    where "REAL_ID" = #{realId,jdbcType=VARCHAR}
  </update>
  <update id="updateByPrimaryKey" parameterType="com.share.golden.entity.UserRealName">
    update USER_REAL_NAME
    set "USER_ID" = #{userId,jdbcType=VARCHAR},
      "REAL_NAME" = #{realName,jdbcType=VARCHAR},
      "GENDER" = #{gender,jdbcType=VARCHAR},
      "IDENTITY_NUM" = #{identityNum,jdbcType=VARCHAR},
      "ATTESTATION_DATE" = #{attestationDate,jdbcType=TIMESTAMP},
      "CHECK" = #{check,jdbcType=DECIMAL},
      "BRITH_DATE" = #{brithDate,jdbcType=TIMESTAMP},
      "PLACE_OF_DOMICILE" = #{placeOfDomicile,jdbcType=VARCHAR},
      "THE _FRONT_OF_ID_CARD" = #{theFrontOfIdCard,jdbcType=VARCHAR},
      "THE _BACK_OF_ID_CARD" = #{theBackOfIdCard,jdbcType=VARCHAR},
      "AUDITOR" = #{auditor,jdbcType=VARCHAR},
      "AUDITOR_ID" = #{auditorId,jdbcType=VARCHAR}
    where "REAL_ID" = #{realId,jdbcType=VARCHAR}
  </update>
</mapper>